### DISCUSSION TITLE
AudioCapture for multichannel recording

### DISCUSSION THREAD

**Discussion by jbscherrer** [bonsai-rx/bonsai#1289] [Q&A]
I'm trying to record four different mono audio channels simultaneously using a Focusrite Scarlett 18i20. I'm currently using the AudioCapture node, but I can't seem to get it to record from more than one channel at a time, and I can't specify which channel to record from either. Any ideas for how to do multichannel recording, or is this just a limitation of Windows MME?

**Comment by bruno-f-cruz** [comment]
If you mic drivers support multichannel they should appear as additional rows or channels in the device node output.

**Reply by jbscherrer** [reply]
Hmm, the node only outputs one channel and one row. I can get all four channels to appear in Ableton, but only when the Scarlett is configured with ASIO drivers. Does AudioCapture support ASIO?
